nss-day-cohort-44/rare-rest-api-malicous-vortices
# Ts view user profiles

# View All User Profiles
## https://github.com/nss-day-cohort-44/rare-rest-malicious-vortices-client/issues/26


## Type of change

# Testing Instructions
`git fetch origin ts-view-user-profiles`
`git checkout ts-view-user-profiles`

User Instructions: 
1. Log out on the DOM if you are currently logged in (if you are logged in as an Author and not an Admin, click refresh and verify that your navbar does NOT contain a `users` tab.
2. Log in as a user who has admin rights.
3. Verify that the navbar DOES contain a `users` tab.

# Checklist:
